generate_token_json.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
```

Comment:
I would rename this file to `build_token_json.js`

Reply:
done